tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -120,6 +120,13 @@ inline nvinfer1::Dims TensorShapeToTrtDims(const TensorShapeType& shape,
   return trt_dims;
 }
 
+inline nvinfer1::Dims TensorShapeArrayToTrtDims(const std::vector<int>& shape,
```

Comment:
can we use TensorShapeArrayToTrtDims overload that returns an error from the other merged PR?
https://github.com/tensorflow/tensorflow/pull/23909/files

Reply:
Thanks Smit, I will rebase.